scripts/standard_configure.sh

```diff
@@ -47,5 +47,5 @@ cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" \
 -DUSE_EIGEN_MKL=OFF                                                 \
 -DKRATOS_GENERATE_PYTHON_STUBS=ON
 
-# Buid
+# Build
```

Comment:
Line 10 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here: